### PR TITLE
[ADD] Langauge translation for  Audit  log  Room  model

### PR DESCRIPTION
**Pull request by jananjoy**

### CODE REVIEW

File: project_resource_calendar/i18n/fr.po [hunk 1]
```diff
@@ -60,6 +70,11 @@ msgid ""
 "the same time."
 msgstr ""
 
+#. module: project_resource_calendar
+#: selection:weekly.report.wizard,state:0
+msgid "Confirmed"
```
Comment (eilst) [inline project_resource_calendar/i18n/fr.po:67]: Add translation: Confirmé

File: project_resource_calendar/i18n/fr.po [hunk 2]
```diff
@@ -47,6 +47,16 @@ msgstr ""
 msgid "Calendar Event Report"
 msgstr ""
 
+#. module: project_resource_calendar
+#: model:ir.ui.view,arch_db:project_resource_calendar.weekly_report_wizard
+msgid "Cancel"
+msgstr ""
```
Comment (eilst) [inline project_resource_calendar/i18n/fr.po]: Add translation if you add string

File: project_resource_calendar/i18n/fr.po [hunk 3]
```diff
@@ -423,72 +438,30 @@ msgid "Service"
...
-#: code:addons/project_resource_calendar/models/calendar_event.py:58
-#: model:ir.ui.view,arch_db:project_resource_calendar.report_events
-#, python-format
-msgid "Tuesday"
-msgstr ""
-
 #. module: project_resource_calendar
 #: model:ir.model.fields,field_description:project_resource_calendar.field_resource_calendar_instrument_category_type
 msgid "Type"
 msgstr "Type"
```
Comment (eilst) [inline project_resource_calendar/i18n/fr.po:523]: And why are you deleting these msgid "Tuesday"	
msgstr "" weekdays?

**Review by eilst — CHANGES_REQUESTED** [review]

File: project_resource_calendar/i18n/fr.po (hunk 1 above)
Comment (jananjoy) [reply project_resource_calendar/i18n/fr.po:67]: @eilst  Done

File: project_resource_calendar/i18n/fr.po (hunk 2 above)
Comment (jananjoy) [reply project_resource_calendar/i18n/fr.po]: @eilst Ok

File: project_resource_calendar/i18n/fr.po (hunk 3 above)
Comment (jananjoy) [reply project_resource_calendar/i18n/fr.po:523]: @eilst  Changed it

**Review by eilst — APPROVED** [review]

**Pull request merged**